renga_deployer/engines.py

```diff
@@ -228,6 +232,10 @@ def launch(self, execution, engine=None, **kwargs):
         """Launch a Kubernetes Job with the context spec."""
         context = execution.context
 
+        execution.environment.update({
+            'DEPLOYER_BASE_URL': '/interactive/{0}'.format(execution.id)
```

Comment:
where is this used?

Reply:
https://github.com/SwissDataScienceCenter/renga-ui/blob/deploy_kube/src/utils/renga-api.ts#L75
It is used in the command of the container, to make it aware of the prefix of its path, dynamically.

Reply:
it should be documented with a default provided in `config.py`